src/portable/raspberrypi/rp2040/dcd_rp2040.c

```diff
@@ -479,4 +473,10 @@ void dcd_edpt_close (uint8_t rhport, uint8_t ep_addr)
 
 }
 
+void dcd_int_handler(uint8_t rhport)
```

Comment:
whoop, I just about to make this change, seems like we have the same idea. This does help with existing cross-platform app.